Markdown/developer_product_guide/fluids.md

```diff
@@ -1,27 +1,25 @@
 # Fluids
 
-[Ansys Common Fluids Format SDK for developers documentation.](https://developer.ansys.com/docs/cff)
+The Ansys Common Fluids Format SDK provides APIs, data models, and example programs to access or write data to Ansys Common Fluids Format files. This format is also used by Ansys Fluids products. For more information, see [Ansys Common Fluids Format SDK for developers (Developer Portal documentation)](https://developer.ansys.com/docs/cff).
 
 ## Fluent
 
-Ansys Fluent  comes with a GUI and a "TUI" (Text User Interface). This is written in a [dialect of Lisp called Scheme](https://en.wikipedia.org/wiki/Scheme_(programming_language)).
+Ansys Fluent includes both a GUI and a TUI. The TUI uses a dialect of Lisp named [Scheme](https://en.wikipedia.org/wiki/Scheme_(programming_language)). For more information about the Fluent TUI, see [Text user interface](https://ansyshelp.ansys.com/public/account/secured?returnurl=/Views/Secured/corp/v242/en/flu_ug/tgd_tcl_tui.html) in the Fluent product documentation.
```

Comment:
```suggestion
Ansys Fluent includes both a GUI and a TUI, which is a user interface that displays text, symbols, and ASCII characters for interaction instead of graphical elements. The TUI uses a dialect of Lisp named [Scheme](https://en.wikipedia.org/wiki/Scheme_(programming_language)). For more information about the Fluent TUI, see [Text user interface](https://ansyshelp.ansys.com/public/account/secured?returnurl=/Views/Secured/corp/v242/en/flu_ug/tgd_tcl_tui.html) in the Fluent product documentation.
```